packages/dapp/src/contexts/Web3Provider.tsx

```diff
@@ -80,6 +81,13 @@ const Web3Provider = ({ children }: { children: any }) => {
     });
   };
 
+  const setTokenContract = (contract: null | any) => {
```

Comment:
You can remove this block now that we're just using setContracts

Reply:
Removed.